homeassistant/components/switch/pilight.py

```diff
@@ -12,10 +13,14 @@
 import homeassistant.components.pilight as pilight
 from homeassistant.components.switch import (SwitchDevice, PLATFORM_SCHEMA)
 from homeassistant.const import (CONF_NAME, CONF_ID, CONF_SWITCHES, CONF_STATE,
-                                 CONF_PROTOCOL)
+                                 CONF_PROTOCOL, STATE_ON)
+from homeassistant.helpers.restore_state import async_get_last_state
 
 _LOGGER = logging.getLogger(__name__)
 
+DOMAIN = 'switch'
```

Comment:
This is a leftover from the entity-id-generation that was there in an earlier version.

Is there any documentation anywhere what entity ids are generated, who is responsible and at which point in time I can be sure that it exist?